objects/obj_main_menu_buttons/Create_0.gml

```diff
@@ -5,7 +5,14 @@ var i;i=-1;
 repeat(8){i+=1;
     hover[i]=0;glow[i]=0;glowing[i]=1;
 }
-
+var path = working_directory+ "JSON_files/gear.JSON"
+show_debug_message("{0}", path);
+var gear =get_open_filename("*.JSON;", "gear.JSON");
+
+show_debug_message("{0}", gear);
+gear = file_text_read_string(gear);
+global.gear = json_decode(gear);
```

Comment:
Just for reference, `json_encode` and `json_decode` are both deprecated, as they return json data in the form of `ds_maps` and `ds_lists` rather than structs. The modern way is to use `json_parse` and `json_stringify` which returns the data in the form of structs and arrays, and then to set the static struct object if the serialized thing has functions that are statically declared.

Here are the docs for reference https://manual.yoyogames.com/#t=GameMaker_Language%2FGML_Reference%2FFile_Handling%2FEncoding_And_Hashing%2Fjson_decode.htm